presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestAllDatatypesFromHiveConnector.java

```diff
@@ -385,4 +486,16 @@ private static TableHandle tableHandleInSchema(TableDefinition tableDefinition)
         }
         return tableHandle;
     }
+
+    public static final class AvroRequirements
```

Comment:
This should be located at the top of the class next to the file type requirements.